docs/howto/system_testing.md

```diff
@@ -143,6 +149,32 @@ data "aws_ami" "latest-amzn" {
 
 Notice the use of the `TEST_RUN_ID` variable. It contains a unique ID, which can help differentiate resources created in potential concurrent test runs.
 
+#### Environment variables
+
+To use environment variables within the Terraform service deployer a `env.yml` file is required.
+
+The file should be structured like this:
+
+```yaml
+version: '2.3'
+services:
+  terraform:
+    environment:
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+```
+
+It's purpose is to inject environment variables in the Terraform service deployer environment.
+
+To specify a default use this syntax: `AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-default}`, replacing `default` with the desired default value.
+
+**NOTE**: Terraform variables passed as environment variables require to be prefixed with `TF_VAR_`. These variables are not available in test case definitions because they are [not injected](https://github.com/elastic/elastic-package/blob/f5312b6022e3527684e591f99e73992a73baafcf/internal/testrunner/runners/system/servicedeployer/terraform_env.go#L43) in the test environment.
+
+#### Cloud Provider access
```

Comment:
Please rename this point as `CI support`. It isn't a mandatory step for regular users, it's more for maintainers.